### DISCUSSION TITLE
Is possible to run in x86_64 arch with macos_instance VMs?

### DISCUSSION THREAD

**Discussion by littleGnAl** [cirruslabs/cirrus-ci-docs#1141] [Q&A] [answered]
Cirrus CI macos_instance runs with arm64 arch now, but my test still needs to run with x86_64 arch to ensure compatibility, is possible to run in x86_64 arch with macos_instance VMs?

**Comment by fkorotkov** [comment, marked as answer]
We removed x86 support on January 1st. You can check [this blog post](https://cirrus-ci.org/blog/2022/11/08/sunsetting-intel-macos-instances/) with reasoning.

As a workaround you can try using Rosetta or run on Linux.

**Comment by littleGnAl** [comment]
Thanks, @fkorotkov , I got it to work by using `arch -x86_64` command.
